src/lazy/binary/raw/v1_1/immutable_buffer.rs

```diff
@@ -394,21 +395,15 @@ impl<'a> BinaryBuffer<'a> {
             EExp(ParseResult<'top, BinaryEExpression_1_1<'top>>),
         }
 
-        use OpcodeType::*;
-        let result = match opcode.opcode_type {
-            EExpressionWith6BitAddress
-            | EExpressionWith12BitAddress
-            | EExpressionWith20BitAddress
-            | EExpressionWithLengthPrefix => {
-                ParseValueExprResult::EExp(self.read_e_expression(opcode))
-            }
-            AnnotationFlexSym | AnnotationSymAddress => {
+        let result = match opcode.kind {
+            OpcodeKind::EExp => ParseValueExprResult::EExp(self.read_e_expression(opcode)),
+            OpcodeKind::Annotations => {
```

Comment:
🗺️ Notice that all of these `match` arms have been consolidated because we now check the `kind` instead of the `opcode_type`.